web/src/components/IdeaList.vue

```diff
@@ -1,34 +1,36 @@
 <template>
   <div class="container-full">
     <div class="container-app">
-      <div id="progress-bar" v-show="filteredIdeas.length !== 0">
-        <div id="progress-bar-progress" :style="{width: ideaProgress + '%'}"></div>
+      <div class="progress-bar" v-show="filteredIdeas.length !== 0">
+        <div class="progress-bar-progress" :style="{width: ideaProgress + '%'}"></div>
       </div>
-      <ul id="ideas">
+      <ul class="ideas">
         <li
           v-for="(idea, index) in filteredIdeas"
           :key="index"
-          :class="{'highlight': index === selectedIndex, 'progress-undecided': idea.progress === 'undecided', 'progress-in-progress': idea.progress === 'in-progress', 'progress-done': idea.progress === 'done'}"
+          :class="{'highlight': index === selectedIndex, 'progress--undecided': idea.progress === 'undecided', 'progress--in-progress': idea.progress === 'in-progress', 'progress--done': idea.progress === 'done'}"
         >
           <div class="idea" @click="notifyIdeaClicked(idea, index)">
             <div>
-              <p id="idea-title" class="text-primary">
+              <p class="idea__title text--primary">
                 {{idea.title}}
                 <span v-if="isNewIdea(idea)">New</span>
               </p>
-              <p id="idea-difficulty" class="badge text-secondary">{{idea.difficulty}}</p>
+              <div>
```

Comment:
Why is there an empty div around the idea difficulty? It does not seem to change the appearance of it.